website/www/site/content/en/contribute/_index.md

```diff
@@ -73,20 +73,33 @@ detail.
 
 ### Prerequisites
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
+ - A GitHub account.
+ - A Linux, macOS, or Microsoft Windows development environment
+      - Required *only* For Windows:
+         - Java JDK 8 installed
+         - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+   changes locally.
+ - For SDK Development:
+      - [Go](https://golang.org) 1.12 or later installed for Go SDK development
  - Python 3.6, 3.7, and 3.8. Yes, you need all three versions installed.
-    - pip, setuptools, virtualenv, and tox installed for Python development
- - for large contributions, a signed [Individual Contributor License
+      - Required *only* For Windows:
+        - pip, setuptools, virtualenv, and tox installed for Python development
```

Comment:
These tools are also needed on linux/mac. Our python tips section asks you to use them
https://cwiki.apache.org/confluence/display/BEAM/Python+Tips

I don't think they were automatically installed. I ran the setup instructions on that page a few months ago, and they worked for me.

Reply:
I've corrected and all prerequisites are needed for linux and mac.